backend/src/main/java/com/cruru/process/controller/ProcessController.java

```diff
@@ -37,6 +40,14 @@ public ResponseEntity<Void> create(
         return ResponseEntity.status(HttpStatus.CREATED).build();
     }
 
+    @PatchMapping("/{process_id}")
+    public ResponseEntity<ProcessResponse> update(
+            @PathVariable(name = "process_id") Long processId,
+            @RequestBody @Valid ProcessUpdateRequest request) {
+        ProcessResponse response = processService.update(processId, request);
```

Comment:
```suggestion
            @RequestBody @Valid ProcessUpdateRequest request
    ) {
        ProcessResponse response = processService.update(processId, request);
```
원래 개행이 컨벤션인데, 위에 create도 개행이 되어 있지 않아 도비가 헷갈렸나보네요 ㅎㅎ. 둘 다 변경 부탁드립니다.